notebooks/102-pytorch-onnx-to-openvino/102-pytorch-onnx-to-openvino.ipynb

```diff
@@ -67,28 +80,36 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "IMAGE_WIDTH = 1024  # Suggested values: 2048, 1024 or 512. The minimum width is 512.\n",
-    "# Set IMAGE_HEIGHT manually for custom input sizes. Minimum height is 512.\n",
-    "IMAGE_HEIGHT = 1024 if IMAGE_WIDTH == 2048 else 512\n",
+    "IMAGE_WIDTH = 780\n",
+    "IMAGE_HEIGHT = 520\n",
     "DIRECTORY_NAME = \"model\"\n",
-    "BASE_MODEL_NAME = DIRECTORY_NAME + f\"/fastseg{IMAGE_WIDTH}\"\n",
-    "\n",
-    "# Paths where PyTorch, ONNX and OpenVINO IR models will be stored.\n",
-    "model_path = Path(BASE_MODEL_NAME).with_suffix(\".pth\")\n",
-    "onnx_path = model_path.with_suffix(\".onnx\")\n",
-    "ir_path = model_path.with_suffix(\".xml\")"
+    "BASE_MODEL_NAME = DIRECTORY_NAME + \"/lraspp_mobilenet_v3_large\"\n",
+    "weights_path = Path(BASE_MODEL_NAME + \".pt\")\n",
+    "\n",
+    "# Paths where ONNX and OpenVINO IR models will be stored.\n",
+    "onnx_path = weights_path.with_suffix('.onnx')\n",
+    "if not onnx_path.parent.exists():\n",
+    "    onnx_path.parent.mkdir()\n",
+    "ir_path = onnx_path.with_suffix(\".xml\")"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "2b33033e",
    "metadata": {
     "id": "u5xKw0hR0jq6"
    },
    "source": [
-    "### Download the Fastseg Model\n",
-    "\n",
-    "Download, load and save the model with pre-trained weights. This may take some time if you have not downloaded the model before."
+    "### Load Model\n",
+    "\n",
+    "Generally, PyTorch models represent an instance of `torch.nn.Module` class, initialized by a state dictionary with model weights.\n",
+    "Typical steps for getting a pretrained model:\n",
+    "1. Create instance of model class\n",
+    "2. Load checkpoint state dict, which contains pretrained model weights\n",
+    "3. Turn model to evaluation for switching some operations to inference mode\n",
+    "    \n",
+    "The `torchvision` module provides a ready to use set of functions for model class initializatio. We will use `torchvision.models.segmentation.lraspp_mobilenet_v3_large`. You can directly pass pretrained model weights to the model initialization function using weights enum `LRASPP_MobileNet_V3_Large_Weights.COCO_WITH_VOC_LABELS_V1`. However, for demonstration purposes we will create it separately. Download pretrained weights and load the model. This may take some time if you have not downloaded the model before."
```

Comment:
```suggestion
    "The `torchvision` module provides a ready to use set of functions for model class initialization. We will use `torchvision.models.segmentation.lraspp_mobilenet_v3_large`. You can directly pass pretrained model weights to the model initialization function using weights enum `LRASPP_MobileNet_V3_Large_Weights.COCO_WITH_VOC_LABELS_V1`. However, for demonstration purposes, we will create it separately. Download pretrained weights and load the model. This may take some time if you have not downloaded the model before."
```